<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>863</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="demosaicMosaic">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gråtonemosaikk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicOriginal">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="demosaicOverskrift">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>761</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <italic>false</italic>
     <bold>true</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="text">
    <string>Demosaicing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>231</width>
     <height>231</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Informasjonen i originalbildet &lt;br/&gt;legges over i en gråtonemosaikk,&lt;br/&gt;før det rekonstrueres ved hjelp&lt;br/&gt;av demosaicing.&lt;br/&gt;&lt;br/&gt;Velg til høyre hvilket bilde som&lt;br/&gt;skal brukes. Under bildet velges&lt;br/&gt;om man vil bruke egenskrevet&lt;br/&gt;kode, eller om man vil bruke&lt;br/&gt;kode med ferdige biblioteker.&lt;/p&gt;&lt;p&gt;Lagre bildet ved å klikke på &lt;br/&gt;lagreknappen nederst til høyre.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicCode">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>390</y>
     <width>111</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Vis kode</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicDemosaic">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>390</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Demosaiced</string>
   </property>
  </widget>
  <widget class="QPushButton" name="adjuster">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>70</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adjuster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="business">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Business</string>
   </property>
  </widget>
  <widget class="QPushButton" name="trees">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>350</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Trees</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stillife">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>310</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Stillife</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mountains">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>230</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Mountains</string>
   </property>
  </widget>
  <widget class="QPushButton" name="balls">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>110</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Balls</string>
   </property>
  </widget>
  <widget class="QPushButton" name="group">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>190</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Group</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ocean">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>270</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ocean</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicDemosaicPackage">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>420</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Demosaiced</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicMosaicPackage">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>420</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Gråtonemosaikk</string>
   </property>
  </widget>
  <widget class="QPushButton" name="demosaicOriginal2">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>420</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Original</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>390</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Uten biblioteker</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>420</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>  Bruk av biblioteker</string>
   </property>
  </widget>
  <widget class="imagewidget" name="imagewidget" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>69</y>
     <width>471</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="save">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>390</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lagre</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>imagewidget</class>
   <extends>QWidget</extends>
   <header>imagewidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
